fuzzers/others/libafl-fuzz/src/executor.rs

```diff
@@ -174,3 +182,57 @@ fn find_executable_in_path(executable: &Path) -> Option<PathBuf> {
         })
     })
 }
+
+pub fn find_afl_binary(filename: &str, same_dir_as: Option<PathBuf>) -> Result<PathBuf, Error> {
+    let is_library =
+        filename.contains('.') && filename.ends_with(".so") || filename.ends_with(".dylib");
```

Comment:
_technically_ you'd want to `to_lowercase()` this, right?
Also, I don't think the `contains` has any meaning here, apart from adding an extra traversal to the filename

Reply:
It's slightly more efficient since if it exits early if there is no `.`
I guess we can `to_lowercase` but whoever has file extensions in uppercase should be banned from operating computers